hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -493,6 +529,75 @@ TypedTokenStore newTokenStore(
 		}
 	}
 
+	private class TransferPrecompile implements Precompile {
+		private SyntheticTxnFactory.FungibleTokenTransfer transferOp;
+
+		@Override
+		public TransactionBody.Builder body(final Bytes input) {
+			transferOp = decoder.decodeTransferToken(input);
+			return syntheticTxnFactory.createCryptoTransfer(List.of(), List.of(), List.of(transferOp));
+		}
+
+		@Override
+		public ExpirableTxnRecord.Builder run(
+				final Address recipient,
+				final Address contract,
+				final WorldLedgers ledgers
+		) {
+			final List<BalanceChange> changes = List.of(
```

Comment:
It's not as obvious as it should be, but the `BalanceChangeManager` constructed [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/grpc/marshalling/ImpliedTransfersMarshal.java#L85) adds assessed fees [to its received `changes` list](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/grpc/marshalling/BalanceChangeManager.java#L66).

So this list must be modifiable; but `List.of()` returns an unmodifiable list.